readme.md

```diff
@@ -1,175 +1,82 @@
+$ cd ~
+$ mkdir src
+$ cd src
+$ git clone https://github.com/Americastestkitchen/nginx-buildpack.git
+$ cd nginx-buildpack
+< make desired changes to scripts/build_nginx>
+$ scripts/build_nginx.sh
 ```
+The above command will create a new binary file at `/tmp/nginx/sbin/nginx`. Copy this file to your `bin` directory in your `nginx-buildback` directory.
 
-**Gemfile**
-```ruby
-source 'https://rubygems.org'
-gem 'unicorn'
-```
-
-**config.ru**
-```ruby
-run Proc.new {[200,{'Content-Type' => 'text/plain'}, ["hello world"]]}
-```
-
-**config/unicorn.rb**
-```ruby
-require 'fileutils'
-preload_app true
-timeout 5
-worker_processes 4
-listen '/tmp/nginx.socket', backlog: 1024
-
-before_fork do |server,worker|
-	FileUtils.touch('/tmp/app-initialized')
-end
-```
-Install Gems
-```bash
-$ bundle install
-```
-Create Procfile
-```
-web: bin/start-nginx bundle exec unicorn -c config/unicorn.rb
-```
-Create & Push Heroku App:
 ```bash
-$ heroku create --buildpack https://github.com/ddollar/heroku-buildpack-multi.git
-$ echo 'https://codon-buildpacks.s3.amazonaws.com/buildpacks/heroku/ruby.tgz' >> .buildpacks
-$ echo 'https://github.com/ryandotsmith/nginx-buildpack.git' >> .buildpacks
-$ git add .
-$ git commit -am "init"
-$ git push heroku master
-$ heroku logs -t
-```
-Visit App
-```
-$ heroku open
-```
-
-## License
-Copyright (c) 2013 Ryan R. Smith
-Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
-The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+$ cp /tmp/nginx/sbin/nginx ~/src/nginx-buildpack/bin/nginx-cedar-14
+```
+
+*NOTE* If you are changing heroku stacks, you will need to name the file based on the name of the heroku stack.
+
+### Test Your New nginx
+1) In the Heroku dashboard, open `atk-kraken-dev`
+2) In the Settings tab, delete the current buildpack
+3) Create a new buildpack, `https://github.com/Americastestkitchen/nginx-buildpack.git#<your-branch-name-here>`
+4) In the Deploy tab, deploy `master` or your current working branch
+5) Test your new build, monitoring the logs for errors
+
+### Deploy your changes to production
```

Comment:
Can we change this to be insanely clear?  Something like:
### Deploy YOUR BRANCH with changes to production

Don't want this to be an "oops" at 3a in an emergency situation